pkg/vm/vm.go

```diff
@@ -537,7 +523,7 @@ func (vm *VirtualMachine) updateFp(instruction *a.Instruction, dstAddr *mem.Memo
 	}
 }
 
-func (vm *VirtualMachine) relocateTrace() []Trace {
+func (vm *VirtualMachine) RelocateTrace() []Trace {
```

Comment:
Since we are making it public I'd suggest adding a description to it

Reply:
done!